randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -116,9 +116,15 @@ Extra - room_id: [3]
           # Upper way
           Screw Attack
   > Other to Entrance Hub
-      Any of the following:
-          Can Break Single Bomb Blocks
-          Speed Booster and Knowledge (Intermediate) and Shinespark Tricks (Advanced) and Disabled Door Lock Rando and Disabled Entrance Rando
+      All of the following:
+          Any of the following:
```

Comment:
since you commented the other block, you can also comment this one with "Destroy the Bomb Block"

Reply:
Makes sense - added the comment